k8s/production/prometheus/release.yaml

```
@@ -149,7 +149,7 @@ spec:
         - pods=[*]
         - nodes=[*]
       metricAnnotationsAllowList:
-        - pods=[gitlab/ci_job_id,metrics*]
+        - pods=[gitlab/ci_job_id,metrics/spack_job_spec_pkg_name,metrics/spack_job_spec_pkg_version,metrics/spack_job_spec_compiler_name,metrics/spack_job_spec_compiler_version,metrics/spack_job_spec_arch,metrics/spack_job_spec_variants,metrics/spack_job_build_jobs]
```

Is it possible to format this like so?
```suggestion
        - > pods=[
                gitlab/ci_job_id,
                metrics/spack_job_spec_pkg_name,
                metrics/spack_job_spec_pkg_version,
                metrics/spack_job_spec_compiler_name,
                metrics/spack_job_spec_compiler_version,
                metrics/spack_job_spec_arch,
                metrics/spack_job_spec_variants,
                metrics/spack_job_build_jobs
            ]
```

I'm not sure if this will work, but according to yaml syntax I _think_ it would.